### PR TITLE
[FEATURE] Google places search

### PR DESCRIPTION
**Pull request by AtherBilal**
### PR Safety Checklist:

 - [x] Added the task to the appropriate release doc under **Enhancements** or **Bug Fixes**
 - [x] Bump `package.json` & `package-lock.json` version numbers to appropriate release
 - [ ] (optional) All external API changes have been documented
 - [ ] (optional) Build docs: `npm run docs`

### Quick Description of Changes (+ screenshots for ui changes):

Added a component to search Google places, then center and zoom to the place, lastly it will add a point on the place

### CODE REVIEW

File: src/GooglePlacesSearch/GooglePlacesSearch.js [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+        </Alert> : null
+      }
+    </div>
+  )
+}
+
+GooglePlacesSearch.propTypes = {
+  /** reference to Openlayers map object */
+  map: PropTypes.object.isRequired,
+  apiKey: PropTypes.string.isRequired
```
Comment (AtherBilal) [inline src/GooglePlacesSearch/GooglePlacesSearch.js:136]: Add comment on how to get API key

File: src/GooglePlacesSearch/GooglePlacesSearch.js [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+  }
+}))
+
+function GooglePlacesSearch (props) {
+  const { map, apiKey } = props
+  const { handleSubmit, register } = useForm()
+  const [errorMessage, setError] = useState(null)
+  const classes = useStyles()
+
+  const DataLoader = (searchString) => {
```
Comment (AtherBilal) [inline src/GooglePlacesSearch/GooglePlacesSearch.js:57]: Change to `dataLoader`

File: src/GooglePlacesSearch/GooglePlacesSearch.js [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+    return fetch(`https://maps.googleapis.com/maps/api/place/findplacefromtext/json?input=${searchString}&inputtype=textquery&fields=geometry&key=${apiKey}`)
```
Comment (AtherBilal) [inline src/GooglePlacesSearch/GooglePlacesSearch.js:58]: Get formatted address for title

File: src/GooglePlacesSearch/GooglePlacesSearch.js (hunk 1 above)
Comment (stazrad) [reply src/GooglePlacesSearch/GooglePlacesSearch.js:142]: nice!

**Pull request merged**